assets/js/atcb.js

```diff
@@ -654,6 +661,9 @@ function atcb_generate_ical(data) {
    }
    if (data['location'] != null && data['location'] != '') {
      ics_lines.push("LOCATION:" + data['location']);
+   }   
+   if (data['recurrence'] != null && data['recurrence'] != '') {
+     ics_lines.push(data['recurrence']);
```

Comment:
Codacy has a fix for the issue: [Replace `·ics_lines.push(data['recurrence'` with `ics_lines.push(data["recurrence"`](https://app.codacy.com/gh/jekuer/add-to-calendar-button/pullRequest?prid=9703945)
```suggestion
    ics_lines.push(data["recurrence"]);
```